Week1/Day2/ExercisesXP/code.py

```diff
@@ -0,0 +1,119 @@
+#Exercise1
+my_fav_numbers = {7,88,24,23,10}
+my_fav_numbers.add(33)
+my_fav_numbers.add(16)
+print(my_fav_numbers)
+my_fav_numbers.remove(10) #we can delete last number only if we know what is last number
+print(my_fav_numbers)
+friend_fav_numbers={21,9,3,15}
+our_fav_numbers = friend_fav_numbers.union(my_fav_numbers)
+print(our_fav_numbers)
+print("---------")
+
+#Exercise2
+sample_tuple = (1,2,3,4,5,6,7,8,9) #it is not possible to add more integers to a tuple
+
+#Exercise3
+basket = ["Banana", "Apples", "Oranges", "Blueberries"]
+basket.remove("Banana")
+basket.remove("Blueberries")
+basket.append("Kiwi")
+basket.insert(0, "Apples")
+print(basket.count("Apples"))
+print(basket)
+print("---------")
+
+#Exercise4
+float = 1.5
+float_list = [float]
+while float < 5:
+    float = float + 0.5
+    float_list.append(float)
+print(float_list)
+print("---------")
+#Exercise5
+for i in range(1,21):
+    print(i)
+print("---------")
+
+for i in range(1,21):
+    if i % 2 == 0:
```

Comment:
is the number the same as his index? what is the first index and what is the first number?

Reply:
no, the first index is 0, the first number is 1